packages/input-group/input-group.scss

```diff
@@ -0,0 +1 @@
+@use "~@fremtind/jkl-core/jkl";
```

Comment:
Først nå legger jeg merke til at denne jo er helt tom.

Skal vi beholde den for concistency og future-proofing, eller skal vi fjerne den?

Reply:
Hmm.. Jeg er jammen ikke sikker 😅 Tror jeg hadde forventet at det ville trengs noen stiler for `fieldset`-varianten, men ser jo ikke ut som det trengs..

Reply:
Jeg valgte heller å duplisere Tooltip-håndteringen. Ikke like DRY, men et fieldset er jo egentlig semantisk en litt annen ting enn en inputgruppe (selv om de visuelt ser veldig like ut).